po/fr.po

```diff
@@ -2722,12 +2707,14 @@ msgstr "Indiquez les permissions du fichier mailbox"
 
 #, c-format
 msgid "%s warning: %s's uid %d is greater than SYS_UID_MAX %d\n"
-msgstr ""
+msgstr "%s attention: l'uid %d de %s est plus grand que SYS_UID_MAX %d\n"
```

Comment:
This line fails because the format changes from `%s %s %d %d` to `%s %d %s %d`.

Reply:
I'd suggest to use the following message:
`%s attention: l'uid de %s, %d, est plus grand que SYS_UID_MAX %d\n`